fullscreen-api/main.css

```diff
@@ -0,0 +1,9 @@
+#contents {
+	width: 600px;
+	font: 14px "Open Sans", sans-serif;
+}
+
```

Comment:
Add a rule to size the video as `max-width: 100%` to make it more responsive on small screens?

Reply:
I added the rule, but it doesn't seem to work when I tested. And I'm not really a css guy. 😅